imap_processing/tests/hi/test_hi_l2.py

```diff
@@ -196,3 +212,14 @@ def test_calculate_ena_intensity(empty_rectangular_map_dataset):
         "ena_intensity_sys_err",
     ]:
         assert var_name in ena_intesity_vars
+
+
+def test_esa_energy_lookup(hi_l1_test_data_path):
+    """Test coverage for esa_energy_lookup()"""
+    lookup_file = (
+        hi_l1_test_data_path / "imap_hi_90sensor-esa-energies_20240101_v001.csv"
```

Comment:
Do you need to mark this as an external test dependency or is it already present in the repo?